src/node-saml/types.ts

```diff
@@ -123,4 +123,5 @@ export interface SamlOptions extends Partial<SamlSigningOptions>, MandatorySamlO
 
   // extras
   disableRequestAcsUrl: boolean;
+  extensions?: any;
```

Comment:
This doesn't help the consumer very much. If there is a way that we could specify something more helpful, that would be good. Even if we just know it will be a string, or an object, that would help. My previous comments about this being XML would be even better if it can be done.

Reply:
I tried `object` type but linting not allow. Need to think on it. Give me some time.